pkg/subctl/components/consts.go

```diff
@@ -20,4 +20,6 @@ const (
 	Connectivity     = "connectivity"
 	ServiceDiscovery = "service-discovery"
 	Globalnet        = "globalnet"
+	Broker           = "broker"
+	Operator         = "Operator"
```

Comment:
For consistency as we don't capitalize the others.

```suggestion
	Operator         = "operator"
```